extension/content_script.js

```diff
@@ -132,19 +118,27 @@ function scroll() {
 	}
 }
 
-// Find start and end of tracker
+// Find start and end of tracker, depending on the type of movement.
+// This method ensures that the following global variables will have been updated:
+// [containerId, container, start, end]
+// Returns: Tracker length, the number of characters being highlighted.
 function setTracker(type) {
-	if (containerInit == 0) { // If no section selected yet, selects first paragraph as container
+	if (containerId === null) {
 		containerId = 0;
-		let container = getContainer(containerId);
+		container = getContainer(containerId);
 		start = 0;
-		setTracker();
```

Comment:
redundant logic w/ move()'s deleted portion. Merged to setTracker().

Reply:
LGTM